### ISSUE TITLE
File upload page doesn't work

### ISSUE THREAD

**Issue by nak3** [dingoblog/dingo#48]
step-1. Go to file upload page - http://localhost:8000/admin/files/

step-2. Choose and upload JPG file. (Push OK)

Result:

- No answer from the server.
- The log output the message as below, but there are no files in the list.

~~~
2016/05/11 - 22:24:39 | 200 |    1.404017ms | ::1 |   POST    /admin/files/upload/
~~~

**Comment by dinever** [comment]
I will take a look at this. Thanks @nak3

**Comment by youngsterxyf** [comment]
@dinever I will try to fix it.

**Comment by dinever** [comment]
Fixed by @youngsterxyf in #68 ⚡
